src/nft/components/profile/list/MarketplaceRow.tsx

```diff
@@ -155,11 +149,11 @@ export const MarketplaceRow = ({
 
   const fees = useMemo(() => {
     if (selectedMarkets.length === 1) {
-      return getRoyalty(selectedMarkets[0], asset) + getMarketplaceFee(selectedMarkets[0], asset)
+      return getRoyalty(selectedMarkets[0], asset) + selectedMarkets[0].fee
     } else {
       let max = 0
       for (const selectedMarket of selectedMarkets) {
-        const fee = getRoyalty(selectedMarket, asset) + getMarketplaceFee(selectedMarket, asset)
+        const fee = getRoyalty(selectedMarket, asset) + selectedMarket.fee
```

Comment:
is there a reason we break this up for if there is only one marketplace - might it be easier just to put it all in the loop?

Reply:
good callout, updated it